tests/Integration/Behaviour/Features/Scenario/Module/module.feature

```diff
@@ -66,10 +67,38 @@ Feature: Module
       | enabled        | true                 |
       | installed      | true                 |
 
+  Scenario: Uninstall modules
+    Given module ps_featuredproducts has following infos:
+      | technical_name | ps_featuredproducts |
+      | version        | 1.0.0               |
+      | enabled        | true                |
+      | installed      | true                |
+    Given module ps_emailsubscription has following infos:
+      | technical_name | ps_emailsubscription |
+      | version        | 1.0.0                |
+      | enabled        | true                 |
+      | installed      | true                 |
+    When I bulk uninstall modules: "ps_featuredproducts,ps_emailsubscription" with deleteFile false
+    Then module ps_featuredproducts has following infos:
+      | installed      | false               |
+    And module ps_emailsubscription has following infos:
+      | installed      | false               |
+
+  Scenario: Uninstall module
+    Given module bankwire has following infos:
+      | technical_name | bankwire            |
+      | version        | 2.0.0               |
+      | enabled        | true                |
+      | installed      | true                |
+    When I uninstall module "bankwire" with deleteFile true
+    And module bankwire has following infos:
+      | installed      | false               |
+    Then I should have an exception that module is not found
```

Comment:
Perfect now we just need one last scenario that tries to uninstall a module not installed already and check that the exception is triggered